### PR TITLE
Add KVM PCI-Passthrough/SR-IOV test config

### PR DESCRIPTION
**Pull request by santwana**
Add KVM PCI-Passthrough/SR-IOV test config

Signed-off-by: Santwana Samantray <santwana@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by sathnaga** [comment]
@santwana add dry-run log...

File: config/tests/guest/libvirt/pci_passthrough.cfg [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+smp = 32
+vcpu_cores = 32
+vcpu_threads = 1
+vcpu_sockets = 1
+# 32G
+mem = 32768
+number_vfs = "Enter the no.of Virtual Function's to be created for each PF"
+# number_vfs = 2
+pf_filter = "ENTER.YOUR.PCI.LABEL"
+# Eg: 0000:05:00.0
```
Comment (sathnaga) [inline config/tests/guest/libvirt/pci_passthrough.cfg]: keep the comment above the param, its little confusing...

File: config/tests/guest/libvirt/pci_passthrough.cfg [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+only (image_backend=filesystem)
+only smallpages
+smp = 32
+vcpu_cores = 32
+vcpu_threads = 1
+vcpu_sockets = 1
+# 32G
+mem = 32768
+number_vfs = "Enter the no.of Virtual Function's to be created for each PF"
+# number_vfs = 2
```
Comment (sathnaga) [inline config/tests/guest/libvirt/pci_passthrough.cfg]: same here

Comment (santwana) [reply config/tests/guest/libvirt/pci_passthrough.cfg]: @sathnaga Addressed the changes, please review.

**Comment by sathnaga** [comment]
@santwana add dry-run result

**Comment by santwana** [comment]
@sathnaga Dry test-run results:
 (1/6) guest_import.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-qemu.unattended_install.import.import.default_install.aio_native: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (2/6) device_passthrough.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.libvirt_pci_passthrough.NIC.SRIOV: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (3/6) device_passthrough.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.libvirt_pci_passthrough.passthrough_reboot.NIC.SRIOV: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (4/6) device_passthrough.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.libvirt_pci_passthrough.passthrough_suspend.NIC.SRIOV: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (5/6) device_passthrough.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.libvirt_pci_passthrough.passthrough_shutdown_start.NIC.SRIOV: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (6/6) guest_remove.qemu.qcow2.virtio_scsi.smp2.virtio_net.Guest.Ubuntu.18.04.ppc64le.powerkvm-libvirt.remove_guest.without_disk: CANCEL: Test cancelled due to --dry-run (0.00 s)

**Pull request merged**